Composer/packages/lib/shared/src/constant.ts

```diff
@@ -173,3 +173,5 @@ export const LgTemplateSamples = {
     };
   },
 };
+
+export const creationFeatureFlagReadMe = 'https://aka.ms/AAabzf9';
```

Comment:
nit: I think resources related to a feature flagged feature should come from the feature code.

Reply:
What do you mean by this? Like the documentation should live near the code in GH? The aka.ms link redirects to a GH file that Andrew is working on for this feature.

Reply:
creationFeatureFlagReadMe variable is defined under lib/shared right now, I think it should be near the creation flow code itself.

Reply:
Right but this constant is used in both the shared and client workspace. Since client references shared and shared does not reference client, this is the only place it can be